framework/contracts/account/src/contract.rs

```diff
@@ -337,18 +335,33 @@ pub fn execute(mut deps: DepsMut, env: Env, info: MessageInfo, msg: ExecuteMsg)
                             .add_messages(msgs),
                     )
                 }
-                ExecuteMsg::ModuleAction { msgs } => {
-                    execute_module_action(deps, info, msgs).map_err(AccountError::from)
-                }
-                ExecuteMsg::ModuleActionWithData { msg } => {
-                    execute_module_action_response(deps, info, msg).map_err(AccountError::from)
-                }
-                ExecuteMsg::IbcAction { msg } => {
-                    execute_ibc_action(deps, info, msg).map_err(AccountError::from)
-                }
-                ExecuteMsg::IcaAction { action_query_msg } => {
-                    ica_action(deps, info, action_query_msg).map_err(AccountError::from)
+
+                // ## Sub-Accounts ##
+                ExecuteMsg::CreateSubAccount {
+                    name,
+                    description,
+                    link,
+                    namespace,
+                    install_modules,
+                    account_id,
+                } => create_sub_account(
+                    deps,
+                    info,
+                    env,
+                    name,
+                    description,
+                    link,
+                    namespace,
+                    install_modules,
+                    account_id,
+                )
+                .map_err(AccountError::from),
+                ExecuteMsg::UpdateSubAccount(action) => {
+                    handle_sub_account_action(deps, info, action).map_err(AccountError::from)
                 }
+
+                // ## Other ##
+                // TODO: Update module migrate logic to not use callback!
```

Comment:
@CyberHoward I don't remember, did you add this TODO here ?

Reply:
I did. In theory we should be able to use the reply payload field.

Reply:
Ok, that's for another PR !

Reply:
*Not without `cosmwasm_2_0` feature